src/main/java/frc/robot/RobotContainer.java

```diff
@@ -510,10 +574,13 @@ private void setupAutChooser() {
         .andThen(new InstantCommand(() -> limelightRotationOff()))
         //Collect HP balls
         .andThen(new DriveOE(drivetrainSubsystem)
-            .andThen(new WaitCommand(1.5))
+            .andThen(new WaitCommand(.75))
             .deadlineWith(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem, colorSensorSubsystem, m_rejectSupplier)))
+        .andThen(new DriveET(drivetrainSubsystem)
+          .andThen(new WaitCommand(1.25))
+          .deadlineWith(new IntakeCommand(intakeSubsystem, centererSubsystem, indexerSubsystem, colorSensorSubsystem, m_rejectSupplier)))
         //Drive to main shoot location
-        .andThen(new DriveEO(drivetrainSubsystem)
+        .andThen(new DriveTO(drivetrainSubsystem)
```

Comment:
Combine the intaking commands and only have one wait:
while Intaking{
  driveOE
  driveET
  Wait 0.75
}
drive TO